JS/src/app/richedit/richedit.component.ts

```diff
@@ -11,48 +8,48 @@ import { AfterViewInit, Component, ElementRef, OnDestroy } from '@angular/core';
   styleUrls: ['./richedit.component.css']
 })
 export class RicheditComponent implements AfterViewInit, OnDestroy {
-  private rich: RichEdit;
+  private rich: RichEdit | null = null;
 
   constructor(private element: ElementRef) { }
 
   ribbonCustomization(options: Options) {
     // remove items
     const fileTab = options.ribbon.getTab(RibbonTabType.File);
-    fileTab.removeItem(FileTabItemId.OpenDocument);
+    fileTab!.removeItem(FileTabItemId.OpenDocument);
 
     const homeTab = options.ribbon.getTab(RibbonTabType.Home);
-    homeTab.removeItem(HomeTabItemId.Copy);
-    homeTab.removeItem(HomeTabItemId.Cut);
-    homeTab.removeItem(HomeTabItemId.Paste);
+    homeTab!.removeItem(HomeTabItemId.Copy);
+    homeTab!.removeItem(HomeTabItemId.Cut);
+    homeTab!.removeItem(HomeTabItemId.Paste);
 
-    homeTab.removeItem(HomeTabItemId.DecreaseFontSize);
-    homeTab.removeItem(HomeTabItemId.IncreaseFontSize);
+    homeTab!.removeItem(HomeTabItemId.DecreaseFontSize);
+    homeTab!.removeItem(HomeTabItemId.IncreaseFontSize);
 
-    homeTab.removeItem(HomeTabItemId.ChangeStyle);
+    homeTab!.removeItem(HomeTabItemId.ChangeStyle);
 
     // remove element from drop down list
```

Comment:
Generally, code comments follow the same grammar rules as the rest of your text. And yes, consistency matters.

```suggestion
    // remove an element from a drop-down list
```